requirements/edx/github.in

```diff
@@ -57,6 +57,7 @@
 -e git+https://github.com/edx/django-wiki.git@1.0.0#egg=django-wiki
 git+https://github.com/edx/MongoDBProxy.git@d92bafe9888d2940f647a7b2b2383b29c752f35a#egg=MongoDBProxy==0.1.0+edx.2
 -e git+https://github.com/jazkarta/edx-jsme.git@690dbf75441fa91c7c4899df0b83d77f7deb5458#egg=edx-jsme
+-e git+https://github.com/technige/py2neo.git@py2neo-3.1.2
```

Comment:
You'll need to run `make upgrade` for this change to propagate to the various `requirements/edx/*.txt` files.  That can be done via devstack or manually triggering https://github.com/edx/edx-platform/actions/workflows/upgrade-python-requirements.yml specifying this PR's branch as the target.  Let me know if you want help with this.

Also, as per the notes at the top of the file, I think we want `#egg=py2neo==3.1.2` appended to the end of this.

Reply:
:+1: